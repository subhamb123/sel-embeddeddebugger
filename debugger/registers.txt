r0:0x00000000FFFFFFFF
r1:0x000000000000002A
r2:0x0000000000000000
r3:0x0000000000000004
r4:0x000000000000000F
r5:0x00000000FFFFFFFF
r6:0x000000000000001C
r7:0x0000000000000002
r8:0x00000000FFFFFFFF
r9:0x0000000000000000
r10:0x00000000FD5C0090
r11:0x0000000000000000
r12:0x0000000000000000
r13:0x0000000000000000
r14:0x0000000000000000
r15:0x0000000000000000
r16:0x0000000000000000
r17:0x0000000000000000
r18:0x0000000000000000
r19:0x0000000000000000
r20:0x0000000000000000
r21:0x0000000000000000
r22:0x0000000000000000
r23:0x0000000000000000
r24:0x0000000000000000
r25:0x0000000000000000
r26:0x0000000000000000
r27:0x0000000000000000
r28:0x0000000000000000
r29:0x0000000000010760
r30:0x000000000000121C
PC:0x00000000000011E0
SP:0x0000000000010760
FPCR:0x00000000
FPSR:0x00000000
MPIDR_EL1:0x80000000
IFSR32_EL2:0x00000000
ESR_EL1:0x00000000
ESR_EL2:0x82BC7626
ESR_EL3:0x1FE00000
ISR_EL1:0x00000000
SCTLR_EL3:0x00C0181D
TCR_EL3:0x80823518
CONTEXTIDR_EL1:0x00000000
CPACR_EL1:0x00000000
ACTLR_EL3:0x00000000
PMCR_EL0:0x41033009
PMCNTENSET_EL0:0x00000000
PMOVSCLR_EL0:0x00000000
PMUSERENR_EL0:0x00000000
PMINTENSET_EL1:0x00000000
PMEVCNTR0_EL0:0x00000000
PMEVCNTR1_EL0:0x00000000
PMEVCNTR2_EL0:0x00000000
PMEVCNTR3_EL0:0x00000000
PMEVCNTR4_EL0:0x00000000
PMEVCNTR5_EL0:0x00000000
PMEVTYPER0_EL0:0x00000000
PMEVTYPER1_EL0:0x00000000
PMEVTYPER2_EL0:0x00000000
PMEVTYPER3_EL0:0x00000000
PMEVTYPER4_EL0:0x00000000
PMEVTYPER5_EL0:0x00000000
PMCCFILTR_EL0:0x00000000
SCR_EL3:0x00000C0E
CPTR_EL3:0x00000000
MDCR_EL3:0x00000000
CNTKCTL_EL1:0x00000000
CNTP_TVAL_EL0:0xB26C5536
CNTP_CTL_EL0:0x00000000
CNTV_TVAL_EL0:0xB0DCD343
CNTV_CTL_EL0:0x00000000
CNTHCTL_EL2:0x00000003
CNTHP_TVAL_EL2:0xA9D45A29
CNTHP_CTL_EL2:0x00000002
CNTPS_TVAL_EL1:0x69C8433E
GICC_CTLR:0x00000000
GICC_PMR:0x00000000
GICC_BPR:0x00000002
GICC_RPR:0x000000FF
GICC_HPPIR:0x000003FF
GICC_ABPR:0x00000003
GICC_AHPPIR:0x000003FF
GICC_APR0:0x00000000
GICC_NSAPR0:0x00000000
GICD_CTLR:0x00000000
GICD_PPISR:0x00000000
GICD_IGROUPR0:0x00000000
GICD_IGROUPR1:0x00000000
GICD_IGROUPR2:0x00000000
GICD_IGROUPR3:0x00000000
GICD_IGROUPR4:0x00000000
GICD_IGROUPR5:0x00000000
GICD_ISENABLER0:0x0000FFFF
GICD_ISENABLER1:0x00000000
GICD_ISENABLER2:0x00000000
GICD_ISENABLER3:0x00000000
GICD_ISENABLER4:0x00000000
GICD_ISENABLER5:0x00000000
GICD_ISPENDR0:0x00000000
GICD_ISPENDR1:0x00000000
GICD_ISPENDR2:0x00000000
GICD_ISPENDR3:0x00000000
GICD_ISPENDR4:0x00000000
GICD_ISPENDR5:0x00000000
GICD_ISACTIVER0:0x00000000
GICD_ISACTIVER1:0x00000000
GICD_ISACTIVER2:0x00000000
GICD_ISACTIVER3:0x00000000
GICD_ISACTIVER4:0x00000000
GICD_ISACTIVER5:0x00000000
GICD_IPRIORITYR0:0x00000000
GICD_IPRIORITYR1:0x00000000
GICD_IPRIORITYR2:0x00000000
GICD_IPRIORITYR3:0x00000000
GICD_IPRIORITYR4:0x00000000
GICD_IPRIORITYR5:0x00000000
GICD_IPRIORITYR6:0x00000000
GICD_IPRIORITYR7:0x00000000
GICD_IPRIORITYR8:0x00000000
GICD_IPRIORITYR9:0x00000000
GICD_IPRIORITYR10:0x00000000
GICD_IPRIORITYR11:0x00000000
GICD_IPRIORITYR12:0x00000000
GICD_IPRIORITYR13:0x00000000
GICD_IPRIORITYR14:0x00000000
GICD_IPRIORITYR15:0x00000000
GICD_IPRIORITYR16:0x00000000
GICD_IPRIORITYR17:0x00000000
GICD_IPRIORITYR18:0x00000000
GICD_IPRIORITYR19:0x00000000
GICD_IPRIORITYR20:0x00000000
GICD_IPRIORITYR21:0x00000000
GICD_IPRIORITYR22:0x00000000
GICD_IPRIORITYR23:0x00000000
GICD_IPRIORITYR24:0x00000000
GICD_IPRIORITYR25:0x00000000
GICD_IPRIORITYR26:0x00000000
GICD_IPRIORITYR27:0x00000000
GICD_IPRIORITYR28:0x00000000
GICD_IPRIORITYR29:0x00000000
GICD_IPRIORITYR30:0x00000000
GICD_IPRIORITYR31:0x00000000
GICD_IPRIORITYR32:0x00000000
GICD_IPRIORITYR33:0x00000000
GICD_IPRIORITYR34:0x00000000
GICD_IPRIORITYR35:0x00000000
GICD_IPRIORITYR36:0x00000000
GICD_IPRIORITYR37:0x00000000
GICD_IPRIORITYR38:0x00000000
GICD_IPRIORITYR39:0x00000000
GICD_IPRIORITYR40:0x00000000
GICD_IPRIORITYR41:0x00000000
GICD_IPRIORITYR42:0x00000000
GICD_IPRIORITYR43:0x00000000
GICD_IPRIORITYR44:0x00000000
GICD_IPRIORITYR45:0x00000000
GICD_IPRIORITYR46:0x00000000
GICD_IPRIORITYR47:0x00000000
GICD_ITARGETSR0:0x01010101
GICD_ITARGETSR1:0x01010101
GICD_ITARGETSR2:0x01010101
GICD_ITARGETSR3:0x01010101
GICD_ITARGETSR4:0x00000000
GICD_ITARGETSR5:0x00000000
GICD_ITARGETSR6:0x01010100
GICD_ITARGETSR7:0x01010101
GICD_ITARGETSR8:0x00000000
GICD_ITARGETSR9:0x00000000
GICD_ITARGETSR10:0x00000000
GICD_ITARGETSR11:0x00000000
GICD_ITARGETSR12:0x00000000
GICD_ITARGETSR13:0x00000000
GICD_ITARGETSR14:0x00000000
GICD_ITARGETSR15:0x00000000
GICD_ITARGETSR16:0x00000000
GICD_ITARGETSR17:0x00000000
GICD_ITARGETSR18:0x00000000
GICD_ITARGETSR19:0x00000000
GICD_ITARGETSR20:0x00000000
GICD_ITARGETSR21:0x00000000
GICD_ITARGETSR22:0x00000000
GICD_ITARGETSR23:0x00000000
GICD_ITARGETSR24:0x00000000
GICD_ITARGETSR25:0x00000000
GICD_ITARGETSR26:0x00000000
GICD_ITARGETSR27:0x00000000
GICD_ITARGETSR28:0x00000000
GICD_ITARGETSR29:0x00000000
GICD_ITARGETSR30:0x00000000
GICD_ITARGETSR31:0x00000000
GICD_ITARGETSR32:0x00000000
GICD_ITARGETSR33:0x00000000
GICD_ITARGETSR34:0x00000000
GICD_ITARGETSR35:0x00000000
GICD_ITARGETSR36:0x00000000
GICD_ITARGETSR37:0x00000000
GICD_ITARGETSR38:0x00000000
GICD_ITARGETSR39:0x00000000
GICD_ITARGETSR40:0x00000000
GICD_ITARGETSR41:0x00000000
GICD_ITARGETSR42:0x00000000
GICD_ITARGETSR43:0x00000000
GICD_ITARGETSR44:0x00000000
GICD_ITARGETSR45:0x00000000
GICD_ITARGETSR46:0x00000000
GICD_ITARGETSR47:0x00000000
GICD_ICFGR0:0xAAAAAAAA
GICD_ICFGR1:0x55540000
GICD_ICFGR2:0x55555555
GICD_ICFGR3:0x55555555
GICD_ICFGR4:0x55555555
GICD_ICFGR5:0x55555555
GICD_ICFGR6:0x55555555
GICD_ICFGR7:0x55555555
GICD_ICFGR8:0x55555555
GICD_ICFGR9:0x55555555
GICD_ICFGR10:0x55555555
GICD_ICFGR11:0x55555555
GICD_SPISR0:0x00000000
GICD_SPISR1:0x00000000
GICD_SPISR2:0x00000000
GICD_SPISR3:0x00000000
GICD_SPISR4:0x00000000
GICD_SPENDSGIR0:0x00000000
GICD_SPENDSGIR1:0x00000000
GICD_SPENDSGIR2:0x00000000
GICD_SPENDSGIR3:0x00000000
FAR_EL3:0x0000000100000000
VBAR_EL3:0x0000000000000000
TTBR0_EL3:0x0000000000005000
MAIR_EL3:0x000000BB0400FF44
AMAIR_EL3:0x0000000000000000
PAR_EL1:0x0000000000000000
TPIDR_EL0:0x0000000000000000
TPIDRRO_EL0:0x0000000000000000
TPIDR_EL1:0x0000000000000000
TPIDR_EL3:0x00000000FFFF8500
RVBAR_EL3:0x0000000000000000
RMR_EL3:0x0000000000000001
SDER32_EL3:0x0000000000000000
CNTFRQ_EL0:0x0000000005F5DD19
CNTVCT_EL0:0x7FFFFFE15637ADC4
CNTP_CVAL_EL0:0x01230CA608A41312
CNTV_CVAL_EL0:0x3F80E000071480F8
CNTVOFF_EL2:0x8000002000001028
CNTHP_CVAL_EL2:0x24004101000C1809
CNTPS_CVAL_EL1:0x82A20002C000011F
CNTPS_CTL_EL1:0x0000000000000000
ELR_EL1:0x0000000000000000
ELR_EL2:0x0980390646088126
ELR_EL3:0x0000000000000D88
v0:0x00000000000106400000000000010640
v1:0x00000000000042900000000000004288
v2:0x00000000000000004197D77464000000
v3:0xA01740A525943D68C5000098A18D1121
v4:0x85284EC445804482118061A9A8406858
v5:0xC1409038069801900185C03800865090
v6:0x00461424B31209E02804028204D0C464
v7:0xA78D907A990290048145702020931055
v8:0x1000081912440948884401014B901011
v9:0x4406682412340C40C85481248492600C
v10:0x88449063810115423D92810991A0C340
v11:0x2A004101405000425420525D003A6920
v12:0xE94004320028050061845CC08D9C4021
v13:0x2064AA7C220887798C30148AD6204C48
v14:0x11190901D69A660B962000392A341407
v15:0xC24C60E00C001A64007AB010CA241080
v16:0xFFFFFF80FFFFFFC80000000000010600
v17:0x1D17430F6A002755B90A105452192448
v18:0xD5640C11B82058D2A7E681490232D620
v19:0x2838480A000A19802414100503686282
v20:0x119693D9C5488880810488008A824E1C
v21:0x8191120A000100C8B670500211485B1A
v22:0x005006C4640C06167140420C13041841
v23:0x094587270908D1650028402028BA5A38
v24:0x89AA42E80886E09094322428C1206C5B
v25:0x42E12750430420700818A0019008020D
v26:0x22A18102000012A4EB7AAA0177808246
v27:0x7214822A0B4B442916C0E35045844966
v28:0x32DAA0C05208B3B18E50308600EC04C4
v29:0x124101A54E34C0C891B16D2DC29087F0
v30:0x0150EB04861284741534A7006802C4C4
v31:0x4E00503D162401604030C6D1E5008808