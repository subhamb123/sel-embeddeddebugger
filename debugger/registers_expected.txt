r0: 00000000ffffffff

r1: 0000000000000000

r2: 0000000000000802

r3: 00000000ff01002c

r4: 0000000000002889

r5: 000000000001b7df

r6: 000000000000000a

r7: 000000000000000a

r8: 000000000000000a

r9: 0000000000000000

r10: 0000000000000001

r11: 0000000000000009

r12: 000000000001f700

r13: 0000000000000000

r14: 0000000000000084

r15: 0000000000000000

r16: 0000000000000000

r17: 0000000000000000

r18: 0000000000000000

r19: 0000000000000000

r20: 0000000000000000

r21: 0000000000000000

r22: 0000000000000000

r23: 0000000000000000

r24: 0000000000000000

r25: 0000000000000000

r26: 0000000000000000

r27: 0000000000000000

r28: 0000000000000000

r29: 000000000001f740

r30: 0000000000001250

pc: 0000000000001254

sp: 000000000001f740

fpcr: 00000000

fpsr: 00000000

mpidr_el1: 0000000080000000

ifsr32_el2: 00000000

esr_el1: 00000000

esr_el2: 02fc7626

esr_el3: 1fe00000

isr_el1: 00000000

sctlr_el3: 00c0181d

tcr_el3: 80823518

contextidr_el1: 00000000

cpacr_el1: 00000000

actlr_el3: 00000000

pmcr_el0: 41033009

pmcntenset_el0: 00000000

pmovsclr_el0: 00000000

pmuserenr_el0: 00000000

pmintenset_el1: 00000000

pmevcntr0_el0: 00000000

pmevcntr1_el0: 00000000

pmevcntr2_el0: 00000000

pmevcntr3_el0: 00000000

pmevcntr4_el0: 00000000

pmevcntr5_el0: 00000000

pmevtyper0_el0: 00000000

pmevtyper1_el0: 00000000

pmevtyper2_el0: 00000000

pmevtyper3_el0: 00000000

pmevtyper4_el0: 00000000

pmevtyper5_el0: 00000000

pmccfiltr_el0: 00000000

scr_el3: 00000c0e

cptr_el3: 00000000

mdcr_el3: 00000000

cntkctl_el1: 00000000

cntp_tval_el0: 71ea1ad3

cntp_ctl_el0: 00000000

cntv_tval_el0: 7051553c

cntv_ctl_el0: 00000000

cnthctl_el2: 00000003

cnthp_tval_el2: 694541e0

cnthp_ctl_el2: 00000002

cntps_tval_el1: 293443c0

gicc_ctlr: 00000000

gicc_pmr: 00000000

gicc_bpr: 00000002

gicc_rpr: 000000ff

gicc_hppir: 000003ff

gicc_abpr: 00000003

gicc_ahppir: 000003ff

gicc_apr0: 00000000

gicc_nsapr0: 00000000

gicd_ctlr: 00000000

gicd_ppisr: 00000000

gicd_igroupr0: 00000000

gicd_igroupr1: 00000000

gicd_igroupr2: 00000000

gicd_igroupr3: 00000000

gicd_igroupr4: 00000000

gicd_igroupr5: 00000000

gicd_isenabler0: 0000ffff

gicd_isenabler1: 00000000

gicd_isenabler2: 00000000

gicd_isenabler3: 00000000

gicd_isenabler4: 00000000

gicd_isenabler5: 00000000

gicd_ispendr0: 00000000

gicd_ispendr1: 00000000

gicd_ispendr2: 00000000

gicd_ispendr3: 00000000

gicd_ispendr4: 00000000

gicd_ispendr5: 00000000

gicd_isactiver0: 00000000

gicd_isactiver1: 00000000

gicd_isactiver2: 00000000

gicd_isactiver3: 00000000

gicd_isactiver4: 00000000

gicd_isactiver5: 00000000

gicd_ipriorityr0: 00000000

gicd_ipriorityr1: 00000000

gicd_ipriorityr2: 00000000

gicd_ipriorityr3: 00000000

gicd_ipriorityr4: 00000000

gicd_ipriorityr5: 00000000

gicd_ipriorityr6: 00000000

gicd_ipriorityr7: 00000000

gicd_ipriorityr8: 00000000

gicd_ipriorityr9: 00000000

gicd_ipriorityr10: 00000000

gicd_ipriorityr11: 00000000

gicd_ipriorityr12: 00000000

gicd_ipriorityr13: 00000000

gicd_ipriorityr14: 00000000

gicd_ipriorityr15: 00000000

gicd_ipriorityr16: 00000000

gicd_ipriorityr17: 00000000

gicd_ipriorityr18: 00000000

gicd_ipriorityr19: 00000000

gicd_ipriorityr20: 00000000

gicd_ipriorityr21: 00000000

gicd_ipriorityr22: 00000000

gicd_ipriorityr23: 00000000

gicd_ipriorityr24: 00000000

gicd_ipriorityr25: 00000000

gicd_ipriorityr26: 00000000

gicd_ipriorityr27: 00000000

gicd_ipriorityr28: 00000000

gicd_ipriorityr29: 00000000

gicd_ipriorityr30: 00000000

gicd_ipriorityr31: 00000000

gicd_ipriorityr32: 00000000

gicd_ipriorityr33: 00000000

gicd_ipriorityr34: 00000000

gicd_ipriorityr35: 00000000

gicd_ipriorityr36: 00000000

gicd_ipriorityr37: 00000000

gicd_ipriorityr38: 00000000

gicd_ipriorityr39: 00000000

gicd_ipriorityr40: 00000000

gicd_ipriorityr41: 00000000

gicd_ipriorityr42: 00000000

gicd_ipriorityr43: 00000000

gicd_ipriorityr44: 00000000

gicd_ipriorityr45: 00000000

gicd_ipriorityr46: 00000000

gicd_ipriorityr47: 00000000

gicd_itargetsr0: 01010101

gicd_itargetsr1: 01010101

gicd_itargetsr2: 01010101

gicd_itargetsr3: 01010101

gicd_itargetsr4: 00000000

gicd_itargetsr5: 00000000

gicd_itargetsr6: 01010100

gicd_itargetsr7: 01010101

gicd_itargetsr8: 00000000

gicd_itargetsr9: 00000000

gicd_itargetsr10: 00000000

gicd_itargetsr11: 00000000

gicd_itargetsr12: 00000000

gicd_itargetsr13: 00000000

gicd_itargetsr14: 00000000

gicd_itargetsr15: 00000000

gicd_itargetsr16: 00000000

gicd_itargetsr17: 00000000

gicd_itargetsr18: 00000000

gicd_itargetsr19: 00000000

gicd_itargetsr20: 00000000

gicd_itargetsr21: 00000000

gicd_itargetsr22: 00000000

gicd_itargetsr23: 00000000

gicd_itargetsr24: 00000000

gicd_itargetsr25: 00000000

gicd_itargetsr26: 00000000

gicd_itargetsr27: 00000000

gicd_itargetsr28: 00000000

gicd_itargetsr29: 00000000

gicd_itargetsr30: 00000000

gicd_itargetsr31: 00000000

gicd_itargetsr32: 00000000

gicd_itargetsr33: 00000000

gicd_itargetsr34: 00000000

gicd_itargetsr35: 00000000

gicd_itargetsr36: 00000000

gicd_itargetsr37: 00000000

gicd_itargetsr38: 00000000

gicd_itargetsr39: 00000000

gicd_itargetsr40: 00000000

gicd_itargetsr41: 00000000

gicd_itargetsr42: 00000000

gicd_itargetsr43: 00000000

gicd_itargetsr44: 00000000

gicd_itargetsr45: 00000000

gicd_itargetsr46: 00000000

gicd_itargetsr47: 00000000

gicd_icfgr0: aaaaaaaa

gicd_icfgr1: 55540000

gicd_icfgr2: 55555555

gicd_icfgr3: 55555555

gicd_icfgr4: 55555555

gicd_icfgr5: 55555555

gicd_icfgr6: 55555555

gicd_icfgr7: 55555555

gicd_icfgr8: 55555555

gicd_icfgr9: 55555555

gicd_icfgr10: 55555555

gicd_icfgr11: 55555555

gicd_spisr0: 00000000

gicd_spisr1: 00000000

gicd_spisr2: 00000000

gicd_spisr3: 00000000

gicd_spisr4: 00000000

gicd_spendsgir0: 00000000

gicd_spendsgir1: 00000000

gicd_spendsgir2: 00000000

gicd_spendsgir3: 00000000

far_el3: 0021c982046c170a

vbar_el3: 0000000000000000

ttbr0_el3: 0000000000014000

mair_el3: 000000bb0400ff44

amair_el3: 0000000000000000

par_el1: 0000000000000000

tpidr_el0: 0000000000000000

tpidrro_el0: 0000000000000000

tpidr_el1: 0000000000000000

tpidr_el3: 00000000ffff8500

rmr_el3: 0000000000000001

sder32_el3: 0000000000000000

cntfrq_el0: 0000000005f5dd19

cntp_cval_el0: 01230c8608a41312

cntv_cval_el0: 3f80e000071080f8

cntvoff_el2: 8140002000001028

cnthp_cval_el2: 24004501000c1809

cntps_cval_el1: 82a20002c000011f

cntps_ctl_el1: 00000000

elr_el1: 0000000000000000

elr_el2: 09c039100600c126

elr_el3: 0000000000003794

v0: 0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a

v1: 0a7325203a7261620000000000000005

v2: 68676665646362610000000000000000

v3: 40000000000000000000000000000000

v4: 00000000000000000000000000000000

v5: 40100401401004014010040140100401

v6: 00000000400000000000000040000000

v7: e786907a992290248105703020938055

v8: 10000c1d1344004888448d0143900011

v9: 4406402092320c40cc54a1240492600c

v10: c8449022810311823d9001099da08100

v11: 6a004101485000421420da5d02394d20

v12: 61400632a028050248845cd08f9c4021

v13: 2864ab7c200887798c3014cad6204848

v14: 11110b01d6ba672b972140393a349407

v15: 0a4460e008001a64005a3410ca241090

v16: ffffff80ffffffc8000000000001f700

v17: 0595070f62702754a90a105650010048

v18: d5640c11b82058d2a7e6a1490232d620

v19: 183c481a020a09802414104103286282

v20: 119293b8c458888485048020aa820e1c

v21: 839130ba08010048b660501215405b1a

v22: 00000644602c07307140420e13041841

v23: 1941072a0900c165082a402028925a38

v24: 81aa42880886e0901436242cc122e45b

v25: 42e12651410421780818a0019088026d

v26: 22a18942440812a4eb5a2a0177808042

v27: 32140328084b442136c1e37045844866

v28: 33daa4c0420833b1865020c600ec1584

v29: b24009906c34c0c895e16f2d829807e0

v30: 0150eb04961280361534e7046800c0c4

v31: 4e09103d062401604030c6d1e4008808